LICENSE

```diff
@@ -0,0 +1,72 @@
+GNU GENERAL PUBLIC LICENSE
```

Comment:
It sounds like we're going to do a dual license.

Please add a simple LICENSE and two more individual files LICENSE.GNU and LICENSE.CC-BY-SA-4.0

I agree with a dual license. I would have done even stricter CC-BY-NC-SA-4.0 but we should leave it CC-BY-SA-4.0 this is better for getting potential future sponsors. eg. Toyota or Softbank or PAL or Fetch want to sell robots, therefor their contributions could be seen as for a commercial purpose- Bigger companies like Google are very strict on letting their employees contribute when there is NC.